marconi-chain-index/test-lib/Gen/Marconi/ChainIndex/Types.hs

```diff
@@ -265,8 +271,9 @@ genProtocolParametersForPlutusScripts =
     <*> CGen.genRational
     <*> pure Nothing -- Obsolete from babbage onwards
     <*> pure (Map.fromList
-      [ (C.AnyPlutusScriptVersion C.PlutusScriptV1, C.CostModel $ fromMaybe (error "Ledger.Params: defaultCostModelParams is broken") defaultCostModelParams)
-      , (C.AnyPlutusScriptVersion C.PlutusScriptV2, C.CostModel $ fromMaybe (error "Ledger.Params: defaultCostModelParams is broken") defaultCostModelParams) ])
+      -- TODO: not sure if it's correct to use `Map.elems` here
```

Comment:
Why would you think it's incorrect? Seems reasonable (especially if it typechecks and all tests pass)

Reply:
It's suspicious to me because we do `Map.toList` and the `CostModel` is a newtype around `[Integer]`. Shouldn't we care about the order of costs?